tasks/celery.yml

```diff
@@ -7,7 +7,7 @@
             mode=0600
   with_items:
     - { name: "default", command: "worker", flags: "{{ celery_worker_extra_args|default('--loglevel=INFO') }}" }
-    - { name: "beat", command: "beat", flags: "--schedule={{ root_dir }}/celerybeat-schedule.db --pidfile={{ root_dir }}/celerybeat.pid --loglevel=INFO" }
+    - { name: "beat", command: "beat", flags: "--schedule={{ root_dir }}/celerybeat-schedule.db --pidfile=/var/run/celerybeat.pid --loglevel=INFO" }
 
```

Comment:
The service doesn't start with these changes. Celery wants  to be run by the project owner, not root, and /var/run/* is owned by root.

Bopping around the internet it looks like the standard way to do this is to create a directory owned by the service in /var/run, owned by the service's user, and to put the pid in that directory. But since /var/run is a temporary filesystem the directory must be created on startup. My suggestion would be:

1. Create a /etc/tmpfile.d/celery.conf rule to create /var/run/celery, owned by the project's user at boot time (and rerun systemd-tmpfile --create when it is installed).
2. Change the line above to put the pidfile into /var/run/celery/celerybeat.pid

I've made these changes on my local tequila-django version of this branch, and that works across reboots. If that sounds like a reasonable change I'll push them up to this branch.

Reply:
secret gist: 
https://gist.github.com/dsummersl/e4ecf35250272e13b00c3475bc8c1d30